src/azure-cli/azure/cli/command_modules/synapse/_help.py

```diff
@@ -226,14 +226,234 @@
 examples:
   - name: Delete a SQL pool.
     text: |-
-        az synapse sql pool delete --name sqlpoolcli1 --workspace-name testsynapseworkspace --resource-group rg
+        az synapse sql pool delete --name sqlpool --workspace-name testsynapseworkspace --resource-group rg
+"""
+
+helps['synapse sql pool restore'] = """
+type: command
+short-summary: Create a new SQL pool by restoring from a backup.
+examples:
+  - name: Create a new SQL pool by restoring an existing SQL pool's restore point.
+    text: |-
+        az synapse sql pool restore --name sqlpool --workspace-name testsynapseworkspace --resource-group rg \\
+        --dest-name newsqlpool --time 2020-11-25T02:47:37
+"""
+
+helps['synapse sql pool show-connection-string'] = """
+type: command
+short-summary: Generates a connection string to a SQL pool.
```

Comment:
>Generates [](start = 15, length = 9)

Generate

Reply:
copied from sql db

---
In reply to: [533204830](https://github.com/Azure/azure-cli/pull/16095#discussion_r533204830) [](ancestors = 533204830)

Reply:
```suggestion
short-summary: Generate a connection string to a SQL pool.
```